lib/web/integrations_awsoidc.go

```diff
@@ -879,3 +880,44 @@ func (h *Handler) awsOIDCConfigureListDatabasesIAM(w http.ResponseWriter, r *htt
 
 	return nil, trace.Wrap(err)
 }
+
+// accessGraphCloudSyncOIDC returns a script that configures the required IAM permissions to sync
+// Cloud resources with Teleport Access Graph.
+func (h *Handler) accessGraphCloudSyncOIDC(w http.ResponseWriter, r *http.Request, p httprouter.Params) (any, error) {
+	queryParams := r.URL.Query()
+
+	cloud := queryParams.Get("provider")
+	switch cloud {
+	case "aws":
```

Comment:
All the other scripts are meant for the aws-oidc integration.
We'll probably need to refactor this when we add other subkind of integrations.
I would just assume that this is for `aws-oidc` and remove this check.

When we add another integrations, we might think that `provider` is not the right wording.
It depends on how things evolve, but we might come up with another `aws` integration that doesn't use IAM IdP w/ OIDC.
So, this `provider` would still be `aws` but we would need another param to actually get the new configuration script.

Reply:
> All the other scripts are meant for the aws-oidc integration. We'll probably need to refactor this when we add other subkind of integrations. I would just assume that this is for `aws-oidc` and remove this check.
> 
> When we add another integrations, we might think that `provider` is not the right wording. It depends on how things evolve, but we might come up with another `aws` integration that doesn't use IAM IdP w/ OIDC. So, this `provider` would still be `aws` but we would need another param to actually get the new configuration script.

I prepared it for the future because we will need TAG GCP/Azure importers soon. 
I don't want to have too many web-endpoints so I multiplexed all of them into a single one

Reply:
Can we use the integration subkind then? `aws-oidc`